androidshared/src/main/java/org/odk/collect/androidshared/ui/Animations.kt

```diff
@@ -0,0 +1,82 @@
+package org.odk.collect.androidshared.ui
+
+import android.animation.Animator
+import android.animation.AnimatorSet
+import android.animation.ValueAnimator
+import android.view.View
+import org.odk.collect.androidshared.ui.Animations.DISABLE_ANIMATIONS
+
+/**
+ * Helpers/extensions for running animations. These are "test safe" in that animations can be disabled
+ * using [DISABLE_ANIMATIONS] - this should be set to `true` in Robolectric tests to avoid
+ * infinite loops.
+ */
+object Animations {
```

Comment:
Not using this at the moment, but I think it'd most likely come in handy later (for any other animations we want) so don't want to get rid of it.